### ISSUE TITLE
Update documentation with examples using JSON

### ISSUE THREAD

**Issue by froehlichcr** [usds/questionable#26]
- [ ] Show data format examples in pure JSON (not TS)
- [ ] Link to core dependencies in README (https://www.npmjs.com/package/use-wizard, https://moment.github.io/luxon/index.html, https://github.com/trussworks/react-uswds)

**Comment by TomNUSDS** [comment]
I put this common on the wrong issue? Making it again here:
> @mchelen-gov pointed out that syntax like ageCalc: (birthday) => !isFraCalculator(birthday, 12) is particularly problematic for json.

**Comment by froehlichcr** [comment]
> I put this common on the wrong issue? Making it again here:
> 
> > @mchelen-gov pointed out that syntax like ageCalc: (birthday) => !isFraCalculator(birthday, 12) is particularly problematic for json.

Yes, the consumer will have to supply the function if required. This is already done in the complex storybook example. If a consumer wants to apply a calculator, they will need to do it client side after the bulk of the JSON has been applied.
